code/modules/antagonists/changeling/powers/tiny_prick.dm

```diff
@@ -183,7 +183,8 @@
 	var/datum/antagonist/changeling/changeling = user.mind.has_antag_datum(/datum/antagonist/changeling)
 	if(!(changeling.has_dna(target.dna)))
 		changeling.add_new_profile(target)
-	return TRUE
+		return TRUE
+	return FALSE
```

Comment:
This is only used to check if we should take away chemicals from changeling and won't fix anything.
This is also unintended buff since it won't take chemicals if we sting someone who's DNA we already have.

Reply:
It's not meant to sting people twice. 
Line 179 is supposed to check that before the target.
`return changeling.can_absorb_dna(target)`

can_absorb_dna should return FALSE if we already have their DNA.
`if(has_dna(target.dna))
	if(verbose)
		to_chat(user, "<span class='warning'>We already have this DNA in storage!</span>")
	return`
But for whatever reason, the sting bypasses this check and sting twice.

Probably has something to do with input delay, and the fact that the ling can technically use his abilities twice per frame since they have no CD...

Reply:
It will still sting twice, just won't take chemicals twice.
This check only checks if we already have the target's DNA to no add it twice.
This is not the fix to a problem, listen to bacon.